openslides/users/static/js/users/pdf.js

```diff
@@ -213,11 +213,11 @@ angular.module('OpenSlidesApp.users.pdf', ['OpenSlidesApp.core.pdf'])
             var createWelcomeText = function() {
                 return [
                     {
-                        text:   getConfigValue('users_pdf_welcometitle'),
+                        text:   gettextCatalog.getString(getConfigValue('users_pdf_welcometitle')),
```

Comment:
Hm. This looks messy. It translates "-" in some circumstances. Better would be to do the translation in the getConfigValue method which can also be renamed to getConfigOrPlaceholer.

Or you just use `Config.translate(Config.get(...).value)` || '-'`